webapp/src/errorList.tsx

```diff
@@ -29,27 +29,37 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
     }
 
     render() {
+        const showAvailableErrors = this.state.errors.length != 0;
         const showCollapseButton = true;
```

Comment:
Instead of "showCollapseButton" being always true, you could make it conditional on available errors.